.gitignore

```diff
@@ -27,6 +27,7 @@
 /v1/backend/config.json.*
 /config.json
 /aws-config.json
+/tests/keepdb
```

Comment:
Rather than using a magic file, can we use an environment variable? The `fs.existsSync(keepFile)` can instead be `_.has(process.env, 'PL_KEEP_TEST_DB')` or something like that.

We should also document this in `https://prairielearn.readthedocs.io/en/latest/dev-guide/#unit-tests-and-integration-tests`

Reply:
I looked for previous examples in PL of environment variables vs magic files (or config.json settings) but didn't really see anything. I liked the file because it had more visibility and permanence in the test directory and was easy to toggle. I definitely want the functionality; I don't really care what triggers it.

+1 for referencing the docs. Thanks for the pointer.

Reply:
I'd rather go with an env var in keeping with the `DEBUG=*` style of requesting different behavior from the running app. I like this because you just set it for a single run, so you are less likely to accidentally leave it lying around. In this case I would just do `PL_KEEP_TEST_DB=yes npm test` whenever I wanted to keep the DB.

Reply:
Works for me.